src/script/interpreter.cpp

```diff
@@ -294,6 +294,7 @@ bool EvalScript(std::vector<std::vector<unsigned char> >& stack, const CScript&
     opcodetype opcode;
     valtype vchPushValue;
     std::vector<bool> vfExec;
+    bool fExec = true;
```

Comment:
By definition, the first opcode must be executed, so `fExec` must be true at the beginning